README.md

```diff
@@ -177,6 +177,84 @@ const document = await bundle(
 console.log(document.json()); // the complete bundled asyncapi document.
 ```
 
+### Resolving external references into components
+**Experimental** 
+
+This is a experimental feature, for now you can resolve external references in messages under `compoents/messages` 
```

Comment:
Any reason you want to call it `experimental` and not just a feature? 🤔

Reply:
At the time I was writing, I thought that since we are only bundling messages into components and in the future we do plan on bundling schemas into schema component and shake down tags etc, this thing would change a lot and go through some breaking changes, so I wrote `experimental`.